app/uploaders/card_image_uploader.rb

```diff
@@ -0,0 +1,21 @@
+class CardImageUploader < CarrierWave::Uploader::Base
+  include CarrierWave::MiniMagick
+
+  storage :file
+
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}"
+  end
+
+  process resize_to_fit: [360,360]
+  process convert: 'jpg'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.